### PR TITLE
disqus integration implemented in articles

### PR DESCRIPTION
**Pull request by ghost**
Even for a minimalistic blog, I needed a comment system so I added disqus to your templates.

### CODE REVIEW

File: templates/article_stub.html [hunk 1]
```diff
@@ -13,7 +13,6 @@ <h2 class="title">
                 <div class="clear"></div>
 
                 <div class="info">
-                    <a href="{{ SITEURL }}/{{ article.url }}">posted at {{ article.date.strftime("%H:%M") }}</a>
```
Comment (jody-frankowski) [inline templates/article_stub.html:16]: Why did you remove this line?

Comment (ghost) [inline templates/article_stub.html:16]: Sorry, it's unrelated to the pull request but I already commited the change before working on the disqus integration.

Comment (jody-frankowski) [inline templates/article_stub.html:16]: If you fix your pr, i'll be happy to merge it :)

**Comment by ghost** [comment]
Everything should be allright now.

**Comment by jody-frankowski** [comment]
Thanks, merged.
